infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/CorfuReplicationClusterManagerAdapter.java

```diff
@@ -35,7 +35,7 @@ public synchronized void updateTopologyConfig(TopologyConfigurationMsg newTopolo
     }
 
     public void prepareSiteRoleChange() {
-        corfuReplicationDiscoveryService.prepareSiteRoleChange();
+        corfuReplicationDiscoveryService.prepareClusterRoleChange();
```

Comment:
I think we have something loose in this API. It does not restrict ClusterRole change from standby -> active or active->standby however, our underlying only process one type. Maybe it's the naming? or revising the actual functionality?